src/coreComponents/denseLinearAlgebra/unitTests/testDenseLASolvers.cpp

```diff
@@ -35,9 +35,9 @@ template< std::ptrdiff_t N >
 class LinearSystem
 {
 public:
-  real64 matrix[N][N];
-  real64 solution[N];
-  real64 rhs[N];
+  real64 matrix[N][N] = {};
+  real64 solution[N] = {};
+  real64 rhs[N] = {};
```

Comment:
```suggestion
  real64 matrix[N][N]{};
  real64 solution[N]{};
  real64 rhs[N] {};
```

it's equivalent but this is how it's done in the rest of the code